.github/workflows/python.yml

```diff
@@ -0,0 +1,163 @@
+name: Python package
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - release-*cli-*
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - x86
+          - x86_64
+    steps:
+      - uses: actions/checkout@v4
+      - if: matrix.target == 'x86_64'
+        name: Build wheels (static openssl)
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.target }}
+          args: --release --out dist --features vendored-openssl
+          sccache: "true"
+          manylinux: auto
+          before-script-linux: |
+            yum install -y perl-IPC-Cmd
```

Comment:
Shouldn't it be gated with something like `command -v yum &>/dev/null && yum install [...]`? As `yum` is only available on RHEL-like distros.
And then, wouldn't it be needed to have the same for apt/apk/dnf/pacman/whatever? :-/

Reply:
Ok nevermind, this is used only in the build process... i thought it was like post-install command of the wheel package.